src/main/java/com/android/volley/CacheDispatcher.java

```diff
@@ -64,6 +71,7 @@ public CacheDispatcher(
         mNetworkQueue = networkQueue;
         mCache = cache;
         mDelivery = delivery;
+	mWaitingRequestHandler = new WaitingRequestHandler(this);
```

Comment:
nit: the indentation here seems to be off by 1 space.

Reply:
Done.